src/repairnator-core/src/main/java/fr/inria/spirals/repairnator/dockerpool/DockerPoolManager.java

```diff
@@ -62,17 +64,30 @@ public void setEngines(List<SerializerEngine> engines) {
      * @param buildId
      * @return
      */
-    public TreatedBuildTracking prepareBeforeSubmitBuild(long buildId) {
```

Comment:
This class should remain independent of the builds

Reply:
Done! This change led me to abstract out some JTravis references too, I think that's a good thing if we are to phase out Travis code.